src/group-configurations/data/slice.js

```diff
@@ -16,12 +21,17 @@ const slice = createSlice({
     updateLoadingStatus: (state, { payload }) => {
       state.loadingStatus = payload.status;
     },
+    updateSavingStatuses: (state, { payload }) => {
+      state.savingStatuses = { ...state.savingStatuses, ...payload };
+    },
   },
 });
 
 export const {
   fetchGroupConfigurations,
   updateLoadingStatus,
+  updateSavingStatuses,
+  updateContentGroup,
```

Comment:
[question]: Do we need `updateContentGroup`?

Reply:
Good catch, thanks